main.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const ball = document.querySelector('.ball');
+
+document.addEventListener('click', event => {
+const ballStyles = getComputedStyle(ball);
+const width = ballStyles.width.replace('px', '');
+const height = ballStyles.height.replace('px', '');
+
+ball.style.left = `${event.clientX - width / 2}px`;
+ball.style.top = `${event.clientY - height / 2}px`;
+
+if (event.clientX + width / 2 > window.innerWidth) {
```

Comment:
destruct properties from the object if you need them more then once(`clientX `, `innerWidth `, etc.)